### ISSUE TITLE
Can't drag the point on curve with finger if it's near the buttons

### ISSUE THREAD

**Issue by nothingshiru** [photopea/photopea#2971]
Maybe the buttons are too close to the curve

https://user-images.githubusercontent.com/79258245/109787861-a3f2fe80-7c49-11eb-803c-f1f9be01e45e.mp4

**Comment by photopea** [comment]
Hi, I increased the tolerance, you do not have to be very precise now. Does it work better?

**Comment by nothingshiru** [comment]
No

**Comment by photopea** [comment]
Now, you have to click at least 15 pixels from the point you want to move. If it still does not work, I think it is a problem in your device. It works well on my phone.

**Comment by nothingshiru** [comment]
It works in another browser but not in Chrome. I can drag the points out.But the buttons are still too easy to be touched in that browser. 
I tried it in my phone too. It works better but there's still a low probability that buttons would be touched. 
Is it related to the size of screen? 

By the way, I have made a image. 
In Chrome, the red area is taken by buttons. Touching them is equivalent to touching buttons. 
![Screenshot_20210304_011527_com](https://user-images.githubusercontent.com/79258245/109850220-3bc60c00-7c8d-11eb-954b-9298423ed4a2.png)
